test/test.report.js

```diff
@@ -185,3 +180,27 @@ tap.test('can pass query params to count', (t) => {
     t.end();
   });
 });
+
+tap.test('can use the report method to get a list of metrics from the db in csv format', (t) => {
+  setup.server.inject({
+    method: 'GET',
+    url: '/api/report.csv'
+  }, (response) => {
+    t.equal(response.statusCode, 200, 'returns HTTP 200');
+    t.equal(typeof response.result, 'string');
```

Comment:
Should we be checking the output too? We aren't really testing the formatting changes you are making